jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -286,6 +294,11 @@ YUI.add('deployment-flow', function() {
         // Here we need to just prevent the deployment flow to close.
         return;
       }
+      this.props.analytics.send(
+        'Deployment Flow',
+        'Button click',
+        'Close - Exit deployment'
```

Comment:
Agreed, standalone it doesn't have much use or much reasoning. In the journey of a bigger set of stats we can answer some of those questions, or at least get an idea. For example if we have a journey like:
Deploy > Add Creds > Cancel > Exit. We can reason adding creds was the blocker.